staker/block_validator.go

```diff
@@ -124,6 +129,7 @@ var DefaultBlockValidatorConfig = BlockValidatorConfig{
 	PendingUpgradeModuleRoot: "latest",
 	FailureIsFatal:           true,
 	Dangerous:                DefaultBlockValidatorDangerousConfig,
+	DataPoster:               dataposter.DefaultDataPosterConfig,
```

Comment:
nit. do we need to add RedisUrl  and RedisLock  as well to DefaultBlockValidatorConfig and TestBlockValidatorConfig ?

Reply:
Added.